src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -6,6 +6,8 @@
 
 # pylint: disable=line-too-long
 
+infrastructure_types = ["generic", "azure", "AWS", "GCP", "Azure_Stack_HC", "Azure_Stack_Hub", "Azure_Stack_Edge", "vSphere"]
```

Comment:
Have following values:

- generic
- AWS
- GCP
- Azure
- AzureStackEdge
- AzureStackHub
- AzureStackHCI
- vSphere

Reply:
vSphere I'll check if we can differentiate between 6.x and 7.x by heuristics